client/src/pages/sign-in.tsx

```diff
@@ -61,7 +72,7 @@ export default function SignIn({ account }: { account: string }) {
             <SingleLineInput
               type="text"
               name="account"
-              label="Email or mobile"
+              label="Username"
```

Comment:
Replace with Email or mobile, "username" is not usually associated with emails or mobile but actual user names like SodaVolcano, can be confusing